osler/templates/core/base.html

```diff
@@ -62,7 +62,7 @@
           {% if settings.OSLER_DISPLAY_ATTENDANCE %}
             <li><a href={{ settings.OSLER_ATTENDANCE_URL }} target="_blank">Attendance</a></li>
           {% endif %}
-          <li><a href="{% url 'about' %}" target="_blank">About</a></li>
+           <li><a href="{% url 'about' %}">{{settings.OSLER_ABOUT_NAME}}</a></li>
```

Comment:
keep target="_blank". This will ensure that it opens up in a new tab, which is what we want